metadata.rb

```diff
@@ -17,6 +19,7 @@
 depends          'percona'
```

Comment:
Please add a lock with ``~> 0.16.1`` which matches what we have in production too.